packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/typings.d.ts

```diff
@@ -1,2 +1,54 @@
-declare module 'pdfjs-dist';
 declare module 'pdfjs-dist/build/pdf.worker.min.js';
+
+//
+// Declare modules and their types that is referred from PDF text layer rendering.
+// Unused properties are commented out
+//
+declare module 'pdfjs-dist/lib/web/ui_utils' {
```

Comment:
Are none of these included in the `@types` pkg you added?

Reply:
Right. The `@types/pdfjs-dist` only includes typings for _core_ classes.